definitions/npm/jasmine_v3.x.x/flow_v0.125.x-/jasmine_v3.x.x.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable flowtype/no-weak-types */
```

Comment:
Not 100% sure about this. It's just a warning, Well those are a lot of `any` (looks like TS) :sweat_smile:

Reply:
This will ends up in installed definitions, so please remove.

Reply:
Sounds like a lint rule against eslint-disable comments is needed.

Reply:
I did a mass change of `Function` to `(...any) => any`.